opencti-platform/opencti-front/src/utils/graph/RelationshipDetails.tsx

```diff
@@ -0,0 +1,226 @@
+import { useFormatter } from '../../components/i18n';
+import { resolveLink } from '../Entity';
+import ExpandableMarkdown from '../../components/ExpandableMarkdown';
+import ItemMarkings from '../../components/ItemMarkings';
+import ItemAuthor from '../../components/ItemAuthor';
+import ItemConfidence from '../../components/ItemConfidence';
+import {
+  RelationshipDetailsQuery,
+} from './__generated__/RelationshipDetailsQuery.graphql';
+import type { SelectedEntity } from './EntitiesDetailsRightBar';
+
+const useStyles = makeStyles < Theme >(() => ({
+  relation: {
+    marginTop: '20px',
+  },
+  label: {
+    marginTop: '20px',
+  },
+  relationTypeLabel: {
+    marginTop: '15px',
+    marginBottom: -1,
+  },
+}));
+
+const relationshipDetailsQuery = graphql`
+    query RelationshipDetailsQuery($id: String!) {
+        stixCoreRelationship(id: $id) {
+            id
+            entity_type
+            description
+            parent_types
+            start_time
+            stop_time
+            created
+            confidence
+            relationship_type
+            from {
+                ... on BasicObject {
+                    id
+                    entity_type
+                    parent_types
+                }
+                ... on BasicRelationship {
+                    id
+                    entity_type
+                    parent_types
+                }
+                ... on StixCoreRelationship {
+                    relationship_type
+                }
+            }
+            to {
+                ... on BasicObject {
+                    id
+                    entity_type
+                    parent_types
+                }
+                ... on BasicRelationship {
+                    id
+                    entity_type
+                    parent_types
+                }
+                ... on StixCoreRelationship {
+                    relationship_type
+                }
+            }
+            created_at
+            createdBy {
+                ... on Identity {
+                    id
+                    name
+                    entity_type
+                }
+            }
+            objectMarking {
+                edges {
+                    node {
+                        id
+                        definition_type
+                        definition
+                        x_opencti_order
+                        x_opencti_color
+                    }
+                }
+            }
+        }
+    }
+`;
+interface RelationshipDetailsComponentProps {
+  queryRef: PreloadedQuery<RelationshipDetailsQuery>
+}
+const RelationshipDetailsComponent: FunctionComponent<RelationshipDetailsComponentProps> = ({ queryRef }) => {
+  const classes = useStyles();
+  const { t, fldt } = useFormatter();
+
+  const entity = usePreloadedQuery<RelationshipDetailsQuery>(relationshipDetailsQuery, queryRef);
+  const { stixCoreRelationship } = entity;
+
+  return (
+    <div className={classes.relation}>
```

Comment:
Same here about the stixCoreRelationship? condition

Reply:
same reason than for stixCoreObject

Reply:
fixed